stylesheets/src/custom.css

```diff
@@ -76,21 +86,24 @@
     color: #fff;
 }
 .firstaide .ui.modal .content {
-    color: #000;
+	color: #000;
 }
 .firstaide .flex-column {
 	display: flex;
 	flex-direction: column;
 }
 .firstaide .flex-justify-center {
-
-    display: flex;
+	display: flex;
 	justify-content: center;
 }
 .firstaide .flex-align-center {
 	display: flex;
 	align-items: center;
 }
+.firstaide .flex-align-start {
+	display: flex;
```

Comment:
What format are you following for CSS? Are you following alphabetical order like "align" must come before " display". If you are not already using it, then make it a practice to use that format. It will help when the CSS file gets large and also for future developers.